ada_2_fhir/zibs2017/payload/zib-plannedcareactivityfortransfer.xsl

```diff
@@ -0,0 +1,432 @@
+                                <xsl:when test="$orderStatus/@code = 'cancelled'">cancelled</xsl:when>
+                                <xsl:when test="$orderStatus/@code = 'aborted'">noshow</xsl:when>
+                                <xsl:when test="$orderStatus/@code = '385644000'">proposed</xsl:when>
+                                <xsl:when test="$orderStatus/@code = '385651009'">arrived</xsl:when>
+                                <xsl:otherwise>
+                                    <xsl:attribute name="value" select="'unknown'"/>
+                                </xsl:otherwise>
+                            </xsl:choose>
+                        </xsl:attribute>
+                        <xsl:call-template name="ext-code-specification-1.0">
+                            <xsl:with-param name="in" select="$orderStatus"/>
+                        </xsl:call-template>
+                    </status>
+                    
+                    <xsl:if test="contact_type">
+                        <appointmentType>
+                            <xsl:call-template name="code-to-CodeableConcept">
+                                <xsl:with-param name="in" select="contact_type"/>
+                                <xsl:with-param name="treatNullFlavorAsCoding" select="true()"/>
+                            </xsl:call-template>
+                        </appointmentType>
+                    </xsl:if>
+                    
+                    <xsl:for-each select="$startDate">
+                        <start>
+                            <xsl:attribute name="value">
+                                <xsl:call-template name="format2FHIRDate">
+                                    <xsl:with-param name="dateTime" select="xs:string(.)"/>
+                                    <xsl:with-param name="dateT" select="$dateT"/>
+                                    <xsl:with-param name="precision">seconds</xsl:with-param>
+                                </xsl:call-template>
+                            </xsl:attribute>
+                        </start>
+                    </xsl:for-each>
+                    <xsl:for-each select="$endDate">
+                        <end>
+                            <xsl:attribute name="value">
+                                <xsl:call-template name="format2FHIRDate">
+                                    <xsl:with-param name="dateTime" select="xs:string(.)"/>
+                                    <xsl:with-param name="dateT" select="$dateT"/>
+                                    <xsl:with-param name="precision">seconds</xsl:with-param>
+                                </xsl:call-template>
+                            </xsl:attribute>
+                        </end>
+                    </xsl:for-each>
+                    
+                    <xsl:for-each select="(reden_contact/afwijkende_uitslag | contact_reason/deviating_result)[@value]">
+                        <reason>
```

Comment:
This section should be moved immediately below `appointmentType`. Make sure to move the `indication` section below as well.

Reply:
Actually weird that the validator didn't see this